modules.json

```diff
@@ -290,6 +290,10 @@
                         "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c",
                         "installed_by": ["modules"]
                     },
+                    "smncopynumbercaller": {
+                        "branch": "master",
+                        "git_sha": "e70433b04bef836b22bbed5a1e6e4e95aeec09fe"
```

Comment:
Did you update the smncopynumber module by hand? The nf-core tools should add `"installed_by": ["modules"]`

Reply:
I used the nf-core commands, but got a linting error for modules.json, so in the end I only copied the updated sha key. I will delete and reinstall smncaller into the branch.